runelite-client/src/main/java/net/runelite/client/plugins/metronome/MetronomePluginConfiguration.java

```diff
@@ -28,29 +28,49 @@
 import net.runelite.client.config.Config;
 import net.runelite.client.config.ConfigGroup;
 import net.runelite.client.config.ConfigItem;
+import net.runelite.client.config.Range;
 
 @ConfigGroup("metronome")
 public interface MetronomePluginConfiguration extends Config
 {
+	int VOLUME_MAX = 127;
```

Comment:
```suggestion
	private static final int VOLUME_MAX = 127;
```

Reply:
Interface fields are always "public", "static" and "final". No change necessary.